Gemfile

```diff
@@ -1,7 +1,4 @@
-source 'https://rubygems.org'
-#this will probably not be available on rubygems, at least initially
-#will probably be only available at https://rubygems.delivery.puppetlabs.net
-#TODO: Change this line when it is actually available internally
+source 'http://rubygems.delivery.puppetlabs.net'
```

Comment:
*shrug* -- just working with what was here. I've changed it back to rubygems.org and deleted the comment